src/devtools-connector/tests/devtools-arc-inspector-test.ts

```diff
@@ -23,7 +22,7 @@ describe('DevtoolsArcInspector', () => {
   before(() => DevtoolsForTests.ensureStub());
   after(() => DevtoolsForTests.reset());
   it('produces PEC Log messages on devtools channel', async () => {
-    const loader = new StubLoader({
+    const loader = new Loader(null, {
```

Comment:
a general question - is this the guideline for all the other tests as well, not using StubLoader?

Reply:
Yes